src/pages/Character/CharacterSheetPage/Tabs/TracksSection/LegacyTrack.tsx

```diff
@@ -29,223 +19,38 @@ export function LegacyTrack(props: LegacyTrackProps) {
   const {
     label,
     value,
-    checkedExperience,
     onValueChange,
-    onExperienceChecked,
     isLegacy,
     onIsLegacyChecked,
   } = props;
 
-  const theme = useTheme();
-  const isMobile = useMediaQuery(theme.breakpoints.down("sm"));
-
-  const checks = [];
-  let checksIndex = 0;
-  let checksValue = 0;
-
-  for (let i = 0; i <= 40; i++) {
-    if (i % 4 === 0 && i !== 0) {
-      checks[checksIndex] = checksValue;
-      checksIndex++;
-      checksValue = 0;
-    }
-
-    if (i < value) {
-      checksValue++;
-    }
-  }
-
-  const labelId = useId();
-  const getValueText = (value: number) => {
-    return `${value} ticks: (${Math.floor(value / 4)} boxes fully filled)`;
-  };
-
-  const announce = useStore((store) => store.appState.announce);
-
   return (
```

Comment:
Reworked legacy tracks to use ProgressTrack and added a separate experience tracker
Reasoning:
- the legacy tracks were a little confusing to use with the checkboxes underneath, and it was very easy to accidentally clear your experience by marking the legacy checkbox which resets the whole tracker. this way, the experience is tracked separately so it is more intentional
- a separate element to track experience is easier to understand and use when advancing characters (IMO) and fits more in line with the wording of the advance moves. 
However, it is visually less similar to the actual starforged character sheets so I am open to suggestions or reverting if needed

Reply:
While I agree with all of your points, I'm personally leaning towards keeping it the prior way. Changing it to how you have it now would require me to write a (quick) script on the db to extract out the XP values, which I don't want to do because I'm spending effort on the other repo. Automatically setting it would fix that issue, but then it becomes unattached from the track itself, which I think is more confusing - new players might be confused as to how XP is earned.

The confirm modal is a great idea!

Once again, fully agree with your points - I'm switching to this in the new version.

![image](https://github.com/user-attachments/assets/e37cbe59-accd-4a2e-b8cd-e420d9b4b2bf)

Reply:
Updated with confirm modal :)